docs/guides/development/python/install-pytorch-ubuntu-22-04/index.md

```diff
@@ -0,0 +1,224 @@
+
+### Prerequisites
+
+To install and run PyTorch comfortably, your system should have:
+
+- **OS**: Ubuntu 22.04 or higher
+- **Python**: 3.8 or higher
+- **Pip**: Latest version recommended
+- **Optional**: CUDA 11.8+ for GPU acceleration
+- A working internet connection
+- Clean Python environment (e.g., virtualenv or conda)
+- At least **4 GB RAM** (8 GB+ preferred for training)
+- No conflicting Python packages
+- At least 3 - 5 GB of free disk space, depending on your python version, virtual environment setup, and additional packages. If working on a minimal or resource-constrained system, consider checking available disk space with:
+
+```command
+    df -h
+```
+
+## Installation Steps
+
+1. Update your system
+
+Once you have taken care of prerequisites, run:
+
+```command
+    sudo apt update && sudo apt upgrade
+```
+
+After running these commands, the system prompts you for confirmation [Y/n] before upgrading to ensure you are aware of resource usage and system changes before continuing. Choose "n" to stop and remedy this situation if you don't have the required resources.
+
+{{< note >}}
+After confirming the upgrade, you may be prompted to restart services or resolve config file changes. These are expected so carefully read and follow the contributor-safe defaults when available.
+{{< /note >}}
+
+2. Services restart
+
+Next you will be prompted to select which services to restart. You can accept the defaults and select any others you know are needed. If you are not sure, skip restarts by selecting Cancel, but it may delay updates taking effect. To navigate inside the Restart Services box:
+
+    **Arrow keys**: Use up/down to move between services
+    **Spacebar**: Toggles selection showing an asterisk [*] if selected and blank if not [ ]
+    **Tab**: Moves the cursor to the *<Ok>* or *<Cancel>* button
+    **Enter**: Confirms your selection (activates the highlighted button)
+
+If you selected **Cancel** (or skipped the restart) the upgraded packages are still installed, but the changes won't take effect until the affected services are manually restarted or the system is rebooted.
+
+{{< note >}}
+Rebooting may seem like a quick fix, but it can obscure errors and delay troubleshooting. Restart services directly unless reboot is explicitly required.
+{{< /note >}}
+
+If you skipped service restarts during the upgrade process, you can apply the updates manually using `systemctl`:
+
+```command
+    sudo systemctl restart <service-name>
+```
+
+    Replacing `<service-name>` with the actual service, such as:
+
+```command
+    sudo systemctl restart ssh
+    sudo systemctl restart samba
+    sudo systemctl restart apache2
+```
+This ensures upgraded packages are used without requiring a full reboot. However, if the installer triggers a service restart (e.g., `systemd`, `udev`, or networking), contributors should:
```

Comment:
Generic comments applicable across the guide:
Please follow the writing guidelines. (e.g.,)
https://developers.google.com/style/abbreviations or https://learn.microsoft.com/en-us/style-guide/word-choice/use-us-spelling-avoid-non-english-words